Sources/Subprocess/Configuration.swift

```diff
@@ -630,12 +651,11 @@ internal func _safelyClose(_ target: _CloseTarget) throws {
     }
 }
 
-/// `IODescriptor` wraps platform-specific `FileDescriptor`,
-/// which is used to establish a connection to the standard input/output (IO)
-/// system during the process of spawning a child process. Unlike `IODescriptor`,
-/// the `IODescriptor` does not support data read/write operations;
-/// its primary function is to facilitate the spawning of child processes
-/// by providing a platform-specific file descriptor.
+/// IODescriptor wraps platform-specific FileDescriptor, which is used to establish a
+/// connection to the standard input/output (IO) system during the process of
+/// spawning a child process. Unlike IODescriptor, the IODescriptor does not support
+/// data read/write operations; its primary function is to facilitate the spawning of
+/// child processes by providing a platform-specific file descriptor.
```

Comment:
```suggestion
/// An IO descriptor wraps platform-specific file descriptor, which establishes a
/// connection to the standard input/output (IO) system during the process of
/// spawning a child process. 
///
/// Unlike a file descriptor, the `IODescriptor` does not support
/// data read/write operations; its primary function is to facilitate the spawning of
/// child processes by providing a platform-specific file descriptor.
```

Looked like there was a copy-paste error in there - is the above detail correct?